src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AssociatedServices.feature

```diff
@@ -252,7 +252,7 @@ Scenario Outline: Associated Service - edit price screen - Flat price Data type
 	Given there are no Service Recipients in the order
 	And the User is presented with the Associated Service edit form for a <ProvisioningType> flat price
 	And fills in the Associated Service edit form with valid data
-	And the quanitity is a decimal
+	And the quantity quanitity is a decimal
```

Comment:
Should `quanitity` be `quantity`?

I think the doubling up of `quantity` might look like a mistake at first to most people. Is there a way of tweaking the wording of the step slightly to make it a bit clearer?

Reply:
Agreed, see latest commit